react/features/base/react/functions.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+
+/**
+ * Returns the field value in a platform generic way.
+ *
+ * @param {Object | string} fieldParameter - The parameter passed through the change event function.
+ * @returns {string}
+ */
+export function getFieldValue(fieldParameter: Object | string) {
```

Comment:
This name is way too generic. Maybe `getEventFieldValue` ?

Reply:
it's intentionally generic, as it is supposed to handle the non event based fields too, so calling them getEventFieldValue would suggest that it only handles those.

Reply:
But it really only works for events. You are not going to use this for regular objects, right?

Reply:
No, it works for events (web style onChange event) and simple strings (mobile style onChangeText event)